release/models/bgp/openconfig-bgp-neighbor.yang

```diff
@@ -475,7 +481,9 @@ submodule openconfig-bgp-neighbor {
       type oc-inet:port-number;
       description
         "Remote port being used by the peer for the TCP session
-        supporting the BGP session";
+        supporting the BGP session.  This may be the same value
+        as the configured neighbor-port or another TCP port if
+        the peer initiated the TCP session.";
```

Comment:
This **is expected to** be the same value
as the configured neighbor-port **if the local device initiated the connection** or another TCP port if
the peer initiated the TCP session."; ?